src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -937,6 +937,7 @@ def process_checkin_group(items: list[CheckinItem]):
     completely serially.
     """
     for item in items:
+        update_check_in_volume(item.ts)
```

Comment:
Since we're doing this here, I think it'd likely be better to just pass all the timestamps to the function in bulk. The single process method can just pass a list of one item.

Actually, it'd be even better to just do this in `process_batch`. Just store all the timestamps after we decode the messages, and then call `bulk_update_check_in_volume` with all of them

Reply:
ah yeah good call